addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -271,6 +271,9 @@ function factory(dependencies) {
         hasThread: attr({
             default: true,
         }),
+        hasScrollAdjust: attr({
+            default: false,
+        }),
```

Comment:
don't think you need this

Reply:
yes I do, otherwise it would be undefined as it is defined only in enterprise

Reply:
Oh I see, this was confusing.
I would rename this one to something like `hasMessageListScrollAdjust`, just so it's more clear what it does. `hasScrollAdjust` it clear enough in context of `MessageList`, somewhat good too in context of `ThreadViewer`, but for chatter it's not.